### PR TITLE
Fix test_network_manager_cloud_setup

### PR DESCRIPTION
**Pull request by narmaku**
This test case was being run on RHEL lower than 8.4 since the exclusion logic was only applying to RHEL 8.4 This needs to be excluded in ANY RHEL version lover than 8.5.

### CODE REVIEW

**Review by jabia99 — APPROVED** [review]
LGTM!

File: test_suite/generic/test_generic.py [hunk 1]
```diff
@@ -586,7 +586,7 @@ def test_eth0_network_adapter_setup(self, host):
                 f'Unexpected device name. Expected: "{device_name}"'
 
     @pytest.mark.run_on(['rhel'])
-    @pytest.mark.exclude_on(['rhel7.9', 'rhel8.4'])
+    @pytest.mark.exclude_on(['<rhel8.5'])
```
Comment (sshmulev) [inline test_suite/generic/test_generic.py:589]: seems like the test "test_network_manager_cloud_setup" passed on rhel-8.4 -  but it should be skipped no?

Comment (sshmulev) [reply test_suite/generic/test_generic.py:589]: I noticed to an issue with the regex of the  exclude_on marker will create a PR to fix it - this is why it didn't skip

Comment (sshmulev) [reply test_suite/generic/test_generic.py:589]: https://github.com/osbuild/cloud-image-val/pull/195

**Review by sshmulev — APPROVED** [review]

**Pull request merged**